euth/phases/contents.py

```diff
@@ -0,0 +1,30 @@
+from django.apps import apps
+
+class PhaseContent():
+    @property
+    def identifier(self):
+        return '{}:{:03d}:{}'.format(self.app, self.weight, self.phase)
```

Comment:
You can also just pass `self` into `format()` and define the position of the values inside the string:
```python
'{s.app}:{s.weight:03d}:{s.phase}'.format(s=self)
```
I like this approach because it's flexible and does not depend on the argument ordering. 😃